src/main/java/com/flightstats/hub/webhook/InternalWebhookClient.java

```diff
@@ -0,0 +1,110 @@
+package com.flightstats.hub.webhook;
+
+import com.flightstats.hub.app.HubHost;
+import com.flightstats.hub.cluster.CuratorCluster;
+import com.flightstats.hub.rest.RestClient;
+import com.flightstats.hub.util.HubUtils;
+import com.google.common.annotations.VisibleForTesting;
+import com.sun.jersey.api.client.Client;
+import com.sun.jersey.api.client.ClientResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.util.*;
```

Comment:
We don't have it written down anywhere but we're trying to migrate away from wildcard imports.

Reply:
OK. I'll figure out where I can turn that off in IntelliJ. Pretty sure it's a setting in there somewhere.

Reply:
Yeah. It was less intuitive to change than I had hoped but i don't remember the details :)